src/coreclr/tools/aot/DependencyGraphViewer/DependencyGraphsForm.cs

```diff
@@ -77,5 +88,43 @@ Graph View
 ";
             MessageBox.Show(helpMessage);
         }
+
+        private void browseButton_Click(object sender, EventArgs e)
+        {
+            openFile();
+        }
+
+        private void openFile()
+        {
+            string argPath = textBox1.Text;
+            if (currentFile is null)
+            {
+                DGMLGraphProcessing dgml = new DGMLGraphProcessing(fileCount);
+                currentFile = dgml;
+                fileCount -= 1;
+                dgml.FindXML(argPath);
+                dgml.Complete += (fileID) =>
```

Comment:
What happens if the file has an error? Will the `Complete` event be fired if the thread crashes or aborts?

Reply:
If you run the project it opens the file explorer, but if you close that window and try clicking on open again, it will show "File is already processing. Please Wait"
Also, the file count starts in -1, it only decreases and is used as a file ID, so I'm confused x.x

Reply:
Changed so function FindXML provides a bool as to whether it successfully opened a file